sdk/keyvault/azure_security_keyvault_keys/benches/benchmarks.rs

```diff
@@ -77,11 +81,171 @@ fn key_operations_benchmark(c: &mut Criterion) {
     });
 }
 
+fn key_2901_benchmark_default(c: &mut Criterion) {
+    const ENV_NAME: &str = "AZURE_KEYVAULT_URL";
```

Comment:
Nit: Might as well define this in the module since it's being used multiple times now. No reason to duplicate it and risk it getting out of sync.